.changeset/shaggy-students-shout.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/lab": patch
+---
+
+Move useFloatingUI from Popper to utils
+Remove Popper
```

Comment:
```suggestion
```
Popper isn't really being removed just the folder and the folder structure doesn't affect our output so I wouldn't mention it.